addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -3434,20 +3436,26 @@ registry.ImageOptimize = ImageHandlerOption.extend({
      * @override
      */
     _computeMaxDisplayWidth() {
-        // TODO: read widths from computed style in case container widths are not default
-        const displayWidth = this._getImg().clientWidth;
-        // If the image is in a column, it might get bigger on smaller screens.
-        // We use col-lg for this in snippets, so they get bigger on the md breakpoint
-        if (this.$target.closest('[class*="col-lg"]').length) {
-            // container and o_container_small have maximum inner width of 690px on the md breakpoint
+        const computedStyles = window.getComputedStyle(this._getImg());
+        const displayWidth = parseInt(computedStyles.width);
+
+        // For the logos we don't want to suggest a width too small
+        if (this.$target.closest('nav').length) {
+            return displayWidth * 3;
+        } else if (this.$target.closest('[class*="col-lg"]').length) {
             if (this.$target.closest('.container, .o_container_small').length) {
-                return Math.min(1920, Math.max(displayWidth, 690));
+                const containerMdMaxWidth = parseInt(computedStyles.getPropertyValue('--o-container-md-max-width'));
```

Comment:
690px before, now 720px... I think you missed the gutter size? Or maybe was this a mistake before?